src/app.js

```diff
@@ -1,21 +1,37 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
+const PORT = process.env.port || 3000;
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  const filePath = normalizedURL.pathname;
+  const fileName = filePath.slice(6) || 'index.html';
```

Comment:
@lilia-obushenko When you use the number `6` here it's hard to understand what it means... It's better to use specific string that you replace, then if someone needs to change path he'll quicker understand how.
```suggestion
  const fileName = filePath.replace(`/file/`, '') || 'index.html';
```

Reply:
on other hand, it can break the path if it contains `/file/` somewhere not in the start... so to be sure we need this regexp, which is not so easy anymore ) 
```suggestion
  const fileName = filePath.replace(/^\/file\//g, '') || 'index.html';
```